src/plugins/intel_npu/src/al/include/intel_npu/al/config/config.hpp

```diff
@@ -202,6 +206,16 @@ enum class OptionMode {
 
 std::string_view stringifyEnum(OptionMode val);
 
+///
+/// CompilerVersion
+///
+
+struct compilerVersion {
+    uint32_t compilerVersion;
```

Comment:
what exactly this mean?

Reply:
Seems compilerVersion always 0, do we plan to use it in the future and how?